src/routes/docs/tooling/mcp/claude/+page.markdoc

```diff
@@ -0,0 +1,73 @@
+---
+layout: article
+title: Claude Desktop
+description: Add the MCP server to Claude Desktop
+---
+
+Learn how you can add the Appwrite MCP server to Claude Desktop.
+
+{% info title="Pre-requisite: Install uv" %}
+
+You must install [uv](https://docs.astral.sh/uv/getting-started/installation/) on your system to use the MCP server.
+
+{% /info %}
+
+{% section #step-1 step=1 title="Add MCP server" %}
+
+In the Claude Desktop app, open the app's **Settings** page (press `CTRL + ,` on Windows or `CMD + ,` on MacOS) and head to the **Developer** tab.
+
+![Claude Settings](/images/docs/mcp/claude/claude-settings.png)
+
+Clicking on the **Edit Config** button will take you to the `claude_desktop_config.json` file. In case the file is missing, please visit the [Model Context Protocol](https://modelcontextprotocol.io/quickstart/user#2-add-the-filesystem-mcp-server) docs.
+
+After opening the file, add the following info:
+
+```json
+{
+    "mcpServers": {
+        "appwrite": {
+            "command": "uvx",
+            "args": [
+                "mcp-server-appwrite",
+                "--databases",
+                "--users"
```

Comment:
I think the default should only have databases.. so no flags required

Reply:
Updated the example to focus on `--users`